Gemfile

```diff
@@ -45,6 +45,7 @@ gem "net-smtp", require: false
 group :development, :test do
   gem "byebug", platforms: %i[mri mingw x64_mingw]
   gem "capybara"
+  gem "capybara-screenshot"
```

Comment:
Its useful tool that makes screenshot (both html and image) of the page that failed so it is easily visible what is wrong.

Reply:
Very nice, do these appear as artefacts in the Github Action?

Reply:
If not, we should get that in along side this so you can see the screenshots in failed Github Action test runs. We can add this to the test.yml after the spec runs to store these artefacts so that you could view them:

```
- name: Archive test failure screenshots
  uses: actions/upload-artifact@v3
  with:
    name: test-failure-screenshots
    path: path/to/screenshots/**/*
```

Then it'd be available to download like in this test run: https://github.com/DFE-Digital/npq-registration/actions/runs/5155930311

I found out how to do this here: https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts